src/routes/FavoritesPage/FavoritesPage.js

```diff
@@ -1,11 +1,67 @@
 import React, { Component } from 'react';
+import PetList from '../../components/PetList';
+import PetsService from '../../services/petsService';
+import { Section } from '../../components/Utils/Utils';
+import UsersService from '../../services/usersService';
+import AuthService from '../../services/authService';
+
+export default class FavoritesPage extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      pets: null,
+      typeOfAnimal: [],
+      breed: [],
+      sex: '',
+      minAge: '',
+      maxAge: '',
+      more: []
+    };
+  }
+
+  componentDidMount() {
+    PetsService.getPets({})
+      .then(pets => {
+        UsersService.getUserFavorites(AuthService.getUserIDFromToken())
+          .then(favorites => {
+            const favoritePets = (pets.filter(elem => {
+              return favorites.includes(elem.PetID);
+            }));
+            this.setState({ pets: favoritePets })
+          })
+          
+      })
+      .catch(error => console.log(error));
+  }
 
-export default class FavoritesPage extends Component {  
   render() {
+    const {
+      pets,
+      typeOfAnimal,
+      breed,
+      sex,
+      minAge,
+      maxAge,
+      more
+    } = this.state;
+
     return (
-      <>
+      <Section>
         <h2>Favorites</h2>
```

Comment:
Maybe change the name to "My Favorite Pets" or something?

Reply:
Also, it might be nice to add some description about what our Favorites page does. Some users might get confused with this page and the pets filter